Added an option to avoid the dependencies check when starting the app

noreiller opened the pull request:
fixes #50

noreiller committed:
Added an option to avoid the dependencies check when starting the app

wdjungst commented:
I like this option but I if we are going to open up options I would rather see them be processed in a loop so we can start to pass more flags like mongo redux etc..

noreiller commented:
You're right. This PR fixed the issue when I was using react-projet, but without any response, I fixed it by importing all the react-project code in my project and modified it to comply my needs.
Anyway, thanks for the response!
